### DISCUSSION TITLE
blog/dn-1/

### DISCUSSION THREAD

**Discussion by giscus** [152334H/152334H.github.io#12] [Announcements]
# blog/dn-1/

To train an AI Text-To-Speech (TTS) model, we’ll need to obtain a Labelled Dataset with two things:

Clean audio files, containing only the voice we’re cloning
The dialogue transcript (text) for each audio file

https://152334h.github.io/blog/dn-1/

**Comment by paulmitalipov** [comment]
Wow, excellent work!

Have you made this dataset (.wav file -> transcripts) available anywhere? I am working on a speech recognition class project and I would love to use Disco Elysium data for it!

**Reply by 152334H** [reply]
email me, 152334h at gmail

**Reply by paulmitalipov** [reply]
Great thanks, just reached out.

**Reply by Dimarom4** [reply]
Can i get this file too?

**Reply by yagee** [reply]
And for me please

**Reply by 152334H** [reply]
Pls ask deep research to find the data for you; I am retired

**Reply by Dimarom4** [reply]
> And for me please

https://github.com/Leerion/discoElysiumExplorer/blob/main/migrations/disco_dialogues.sql
